libc/src/__support/File/file.h

```diff
@@ -300,9 +300,11 @@ class File {
   }
 };
 
-// The implementaiton of this function is provided by the platform_file
-// library.
+// The implementation of the following functions is provided by the
+// platform_file library.
 ErrorOr<File *> openfile(const char *path, const char *mode);
+// Create a File object and associate it with a fd.
+ErrorOr<File *> create_file_from_fd(int fd, const char *mode);
```

Comment:
this shouldn't be in the top level `file.h` since it's linux specific. It should be in the linux specific `file.h`

Reply:
If so, I hava to include `linux/file.h` in `fdopen.cpp`, is that OK?

Reply:
Fixed.

Reply:
yes that's fine. Given that this is a linux specific implementation of fdopen I'm okay with it directly including the linux specific file.